src/sgame/sg_admin.cpp

```diff
@@ -5579,135 +5579,128 @@ void G_admin_cleanup()
 	g_admin_commands = nullptr;
 }
 
-bool G_admin_bot( gentity_t *ent )
-{
-	int min_args = 3;
-	char arg1[MAX_TOKEN_CHARS];
-	char name[MAX_TOKEN_CHARS];
-	char team[MAX_TOKEN_CHARS];
-	char skill[MAX_TOKEN_CHARS];
-	char err[MAX_STRING_CHARS];
-	char behavior[MAX_QPATH];
-	int skill_int;
-	int i;
-	int clientNum;
+static void BotUsage( gentity_t *ent ) {
+	static const char bot_usage[] = QQ( N_( "^3bot: ^7usage: bot add (<name> | *) (aliens | humans) [<skill level> [<behavior>]]\n"
+	                                        "            bot del (<name> | all)\n"
+	                                        "            bot names (aliens | humans) <names>…\n"
+	                                        "            bot names (clear | list)" ) );
+	ADMP( bot_usage );
+}
 
-	static const char bot_usage[] = QQ( N_( "^3bot: ^7usage: bot add [^5name|*^7] [^5aliens|humans^7] (^5skill^7) (^5behavior^7)\n"
-	                                        "            bot del [^5name|all^7]\n"
-	                                        "            bot names [^5aliens|humans^7] [^5names…^7]\n"
-	                                        "            bot names [^5clear|list^7]" ) );
+static team_t TeamFromString2( const char* s ) {
+	// exact match only
+	if ( !Q_stricmp( s, "a" ) || !Q_stricmp( s, "aliens" ) )
+	{
+		return team_t::TEAM_ALIENS;
+	}
+	if ( !Q_stricmp( s, "h" ) || !Q_stricmp( s, "humans" ) )
+	{
+		return team_t::TEAM_HUMANS;
+	}
+	return team_t::TEAM_NONE;
+}
 
-	if ( trap_Argc() < min_args )
+static bool BotAddCmd( gentity_t *ent, const Cmd::Args& args ) {
```

Comment:
Please be consistent about where you put the * and &. Current code favors putting the &/* on the variable and not the type.